dialogue-core/src/test/java/com/palantir/dialogue/core/DialogueChannelTest.java

```diff
@@ -301,6 +305,39 @@ void nice_tostring() {
                 .isNotEqualTo(channel2.toString());
     }
 
+    @ParameterizedTest
+    @EnumSource(NodeSelectionStrategy.class)
+    public void test_can_use_sticky_attachments(NodeSelectionStrategy nodeSelectionStrategy) throws ExecutionException {
```

Comment:
Ha if only, forgot the NodeSelectionChannel just sidesteps the strategies for one url.